src/components/inputs/SuggestSelect.jsx

```diff
@@ -387,6 +387,7 @@ const SuggestSelect = React.createClass({
 
   onOptionSelect(option, e) {
     this.selectOption(option.id);
+    if (!this.isMounted()) e.stopPropagation();
```

Comment:
Done: we now use a `_mounted` attribute, as specified in the blog post. 

I still don't understand why `react` doesn't differentiate between an internal (private) `_isMounted` used by React and an external (public) `isMounted` that does what I've done here (keeping track of mount state using react lifecycle callbacks). 

In any case as the blog post says, the `isMounted` method is no longer available in ES6 React components, so this is definitely something we don't want to keep doing.